clubs_guides/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+from factory import Faker
+from django.test import TestCase
+
+from clubs_guides.tests.test_factory import ClubsGuideFactory, CategoryFactory
+from clubs_guides.models import Category
+
+
+class TestCategoryListView(TestCase):
+    def setUp(self):
+        self.categories = []
+        for i in range(2):
```

Comment:
does it make sense to use a numbered array here, instead of a dictionary/map, based on `for key in dict`?